### ISSUE TITLE
каузатив на -т и основы ис(т)-, сис(т)-

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#202]
Аня: Нужно было бы кстати сделать в словаре следующую правку в поле Deriv
Каузативный суффикс =ТЫр переделать на =ДЫр
Каузативный суффикс =(Ы)Т переделать на =т
эм=ДЫр=т-
Точнее, оставить ТЫр, но второй каузатив сделать маленьким т и без вставной гласной.

**Comment by veramal** [comment]
Предлагаю также ввести каузатив на -т в парсер. Есть пара примеров его употребления после КлА:
_Чарабас, чарабас **пасхладарға**._ - Не надо... не надо, чтоб топтали. (В.Г. Шулбаева. Сыын мӱӱзі)
(пас 'наступать'+КлА-т-Fut-Dat)
_ыстастырып, **сорхлада** тартынып, Алтын Табатай одырхан тыты кимірбзе пиргенде_ - принюхавшись, носом воздух втянула, лиственницу, на которой Алтын-Табатай сидел, грызть когда стала
(сор 'сосать'+КлА+т+Cv.a)
Ну и для других случаев пригодится.

**Comment by adybo** [comment]
Давайте введем, от многослогов он любит

**Comment by veramal** [comment]
Добавила в таблицу каузатив т, нужно теперь ввести в автомат.

**Comment by tuffnatty** [comment]
Если я добавляю Сaus _т_, появляется запрещенный когда-то в тестах разбор исткен < изерге

**Comment by adybo** [comment]
Думаю, что каузатив -т не бывает у односложных глаголов

**Comment by veramal** [comment]
Надо, видимо, делать словарную помету, запрещающую каузатив на -т.

**Comment by adybo** [comment]
Там все может быть не так просто, в разных диалектах могли прижиться разные каузативы (сейчас вот в стословниках вроде сагайцы больше любят койдир, а качинцы койгиз жечь)

**Comment by veramal** [comment]
Так, с _исткен_ все проще решается: каузатив т может образовываться только от основ на гласную и на неносовые сонорные _р,л,й_. Не знаю, куда это вписать в документ, но надо учесть.

**Comment by adybo** [comment]
Поэтому и аттар так не может разбираться

**Comment by tuffnatty** [comment]
Тест исткен < изерге по-прежнему сломан

**Comment by veramal** [comment]
Это хорошо.

**Comment by tuffnatty** [comment]
сломан в том смысле, что этот разбор должен быть запрещен, а он не запрещен и возникает.

**Comment by veramal** [comment]
Так а почему? Было ли реализовано вот это: "каузатив т может образовываться только от основ на гласную и на неносовые сонорные р,л,й. Не знаю, куда это вписать в документ, но надо учесть." ?

**Comment by tuffnatty** [comment]
потому что я не занимаюсь вписыванием в документ, а занимаюсь имплементацией этого документа в коде.

**Comment by veramal** [comment]
Ну ок, написала правило 36 по этому поводу.

**Comment by tuffnatty** [comment]
Добавил, тест починился

**Comment by veramal** [comment]
Но тут нам не надо всю систему описывать, а только запретить *койт-.

> Там все может быть не так просто, в разных диалектах могли прижиться
> разные каузативы (сейчас вот в стословниках вроде сагайцы больше любят
> койдир, а качинцы койгиз жечь)

**Comment by veramal** [comment]
_Исткен_ сейчас разбирается как форма от _изерге_ 'слышать' без каузатива, а должен быть сломан. _Пасхладарға_, _сорхлада_ разбираются как надо.

**Comment by tuffnatty** [comment]
> _Исткен_ сейчас разбирается как форма от _изерге_ 'слышать' без каузатива, а должен быть сломан.

а по какому правилу он должен быть сломан?

**Comment by veramal** [comment]
Это правила сандхи в конце документа, стр. 56, раздел III.2. Выбор варианта основы у слов, оканчивающихся на -ст, -ц

**Comment by tuffnatty** [comment]
А в #96 эта форма _исткен_ вроде была допустима? Если добавить это правило как обязательное, то перестает работать тест на _исткен_, добавленный по итогам #96.

**Comment by veramal** [comment]
Не помню, почему там была допустима (видимо, не мешала), но в корпусе нет ни одного примера на _исткен_, тогда как с началом _искен_ 220. Надо сломать.
